src/app/store/effects/VehicleTechRecordModel.effects.ts

```diff
@@ -48,6 +48,8 @@ export class VehicleTechRecordModelEffects {
       EVehicleTechRecordModelActions.GetVehicleTechRecordModelHavingStatusAll
     ),
     map((action) => action.payload),
+    // TODO: Needs refactoring with forkJoin or mergeMap to avoid nesting switchMap
```

Comment:
yes the forkjoin was creating to search for 2 backend call, vin and vrm, on failure we are creating a skeletal scheme to be used for creation. lets remove this comments

Reply:
Happy to remove but we might want to log a ticket, at the moment it is fine but we want to avoid nesting Observables when we have utilities available that allow us to deal the multiples streams.